<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diabtes</class>
 <widget class="QWidget" name="diabtes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #d2f7ff;</string>
  </property>
  <widget class="QTextEdit" name="inputPregnancies">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputGlucose">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputBP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputSkin">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputInsulin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputBMI">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputDPF">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputAge">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3.5px;
 border: 2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #50dfff;
border-radius:3.5px;
 border: 2px solid #43bcd4;</string>
   </property>
   <property name="text">
    <string>predict</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de grossesses</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau de glucose (mg/dL)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Épaisseur de la peau (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Facteur héréditaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IMC (BMI)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau d'insuline</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pression artérielle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #50dfff;
border-radius:3.5px;
 border: 2px solid #43bcd4;
font: 12px Arial;
height: 800px;
Width: 200px;</string>
   </property>
   <property name="text">
    <string>la resultat est: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #50dfff;
border-radius:3.5px;
 border: 2px solid #43bcd4;</string>
   </property>
   <property name="text">
    <string>enquêter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
